packages/plugins/sentry/src/index.ts

```diff
@@ -131,56 +132,93 @@ export const useSentry = <PluginContext extends Record<string, any> = {}>(
         ...addedTags,
       };
 
-      let rootSpan: Span | undefined;
+      const createExecuteFn = async (
```

Comment:
The name of this function is a bit misleading.

What do you think of `executeWithSpan` ?
```suggestion
      const executeWithSpan = async (
```